javalib/src/main/scala/java/util/concurrent/atomic/AtomicIntegerArray.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.util.concurrent.atomic
+
+class AtomicIntegerArray(length: Int) extends Serializable {
```

Comment:
```suggestion
class AtomicIntegerArray(length: Int) extends java.io.Serializable {
```
Otherwise it refers to `scala.Serializable` in older Scala versions.